From db78482907638495e74392250b9b76c0114af75f Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Tue, 3 Sep 2024 20:52:30 +0200
Subject: [PATCH] Fix build on Linux

Was broken in d2415093daabdec8f1622bd11f6fa185d857ecab

buho/src/utils/server.cpp:10:10: fatal error: buhointerface.h: No such file or directory
---
 src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 068129d..2d313b4 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -57,6 +57,7 @@ else()
     add_executable(${PROJECT_NAME}
         ${buho_SRCS}
         ${buho_HDRS}
+        ${buho_dbus_SRCS}
         ${buho_ASSETS})
 endif()
 
-- 
GitLab

